CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### To be Released
 
+* fix(one-off): remove async field from the run command ([PR#1060](https://github.com/Scalingo/cli/pull/1060))
```

Comment:
issue(blocking): I'm not sure we removed the async one-off feature here. But we did remove the `async` params as it is the default behavior.

Reply:
Yeah :/

Reply:
Is it not clear enough?

Reply:
My bad, I missed the `field` word 🤦
It's okay!